index.html

```diff
@@ -45,10 +45,246 @@ <h1 class="hero__heading poppins">
+                <li class="poppins">javaScript</li>
+            </ul>
+
+            <div>
+                <button type='button' class="project__button poppins">
+                    see Project
+                </button>
+            </div>
+
+
+
+
+        </div>
+
+        <div class="project">
+
+            <img src="images/professional_printing.svg" alt="">
+
+
+            <h1 class="project__heading poppins">
+                Tonic
+            </h1>
+
+            <ul class="project__path poppins">
+                <li class="project__path__name poppins">CANOPY</li>
+                <li class="project__path__type poppins">BACK END DEV</li>
+                <li class="project__path__type poppins">2015</li>
+            </ul>
+
+            <p class="project__desc poppins">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+
+            <ul class="project__language">
+                <li class="poppins">html</li>
+                <li class="poppins">css</li>
+                <li class="poppins">javaScript</li>
+            </ul>
+
+            <div>
+                <button type='button' class="project__button poppins">
+                    see Project
+                </button>
+            </div>
+
+
+
+
+        </div>
+
+
+    </section>
+
+
+    <!-- about us -->
+    <section id="aboutus">
+        <div class="aboutus">
+            <h1 class="aboutus__heading poppins">
+                About<br>
+                Myself
+            </h1>
+            <p class="aboutus__desc poppins">
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+                my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.
+            </p>
+            <a class="aboutus__subtitle poppins" href="#">
+                LET’S CONNECT
+            </a>
+            <ul class="aboutus__social">
+                <li> <a class="aboutus__social__link" href="https://twitter.com/ahmed_najash"><img
+                            src="images/Twitter.svg" alt="twitter"></a></li>
+                <li> <a class="aboutus__social__link" href="#"><img src="images/M.svg" alt="M"></a></li>
+                <li> <a class="aboutus__social__link" href="https://www.linkedin.com/in/ahmed-najash-286a671ab/"><img
+                            src="images/Linkedin_icon.svg" alt="icon"></a></li>
+                <li> <a class="aboutus__social__link" href="https://github.com/jash-debug"><img src="images/github.svg"
+                            alt=""></a></li>
+                <li> <a class="aboutus__social__link" href="#"><img src="images/Hand.svg" alt="hand"></a></li>
+            </ul>
+
+            <div>
+                <button type='button' class="aboutus__button poppins">
+                    see Project
+                </button>
```

Comment:
You have done a good job at making an attempt to implement the `button-pressed` interaction of the buttons. However, there's more. The text and background colors don't match that given by your chosen template.

Kindly implement all the button interactions given by the Figma design of your chosen template and make sure their sizes match too. Sticking to the design as much as possible (e.g. font, colors, images, text, margins) will be a good habit right now. You'll get more prepared for a typical workplace experience where you might be required to reproduce a given design as strictly as possible.

Follow the original vision if it's pretty and works, it will be a wonderful website.

Your button-pressed design:
![image](https://user-images.githubusercontent.com/20554583/185748871-79dcea78-c4de-405a-a5f2-2cde5aae8b25.png)

Figma button interactions:
![image](https://user-images.githubusercontent.com/20554583/185748935-6c5b6edf-db95-4c93-b8a2-2e181fbad6a8.png)